### ISSUE TITLE
Zathura exits on startup

### ISSUE THREAD

**Issue by florianbegusch** [netblue30/firejail#3280]
* OS: Arch Linux
* kernel: Linux 5.4.25.a-2-hardened x86_64
* firejail version: build from source, commit: 25796cc5fac4d9460357d2fd9844ce746b4b7a4f

```shell
$ zathura
Reading profile /etc/firejail/zathura.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-exec.inc
Reading profile /etc/firejail/disable-interpreters.inc
Reading profile /etc/firejail/disable-passwdmgr.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/disable-xdg.inc
Reading profile /etc/firejail/whitelist-usr-share-common.inc
Reading profile /etc/firejail/whitelist-var-common.inc
Parent pid 87255, child pid 87305
1 program installed in 1.58 ms
Standard C library installed in 29.89 ms
Program libraries installed in 454.40 ms
GdkPixbuf installed in 142.65 ms
GTK3 installed in 23.76 ms
Pango installed in 0.00 ms
GIO installed in 8.24 ms
Installed 167 libraries and 6 directories
Warning: skipping alternatives for private /etc
Private /etc installed in 7.33 ms
Warning: skipping alternatives for private /usr/etc
Warning: skipping fonts for private /usr/etc
Warning: skipping ld.so.cache for private /usr/etc
Warning: skipping ld.so.conf for private /usr/etc
Warning: skipping ld.so.conf.d for private /usr/etc
Warning: skipping ld.so.preload for private /usr/etc
Warning: skipping machine-id for private /usr/etc
Private /usr/etc installed in 0.13 ms
Warning: /sbin directory link was not blacklisted
Warning: /usr/sbin directory link was not blacklisted
Error open: fs.c:482 fs_remount_simple: No such file or directory
Error: proc 87255 cannot sync with peer: unexpected EOF
Peer 87305 unexpectedly exited with status
```

**Comment by florianbegusch** [comment]
Disabling "disable-common.inc" in zathura's profile solved the issue. I did not look at this any further.

Cheers,
Florian

**Comment by florianbegusch** [comment]
> Disabling "disable-common.inc" in zathura's profile solved the issue. I did not look at this any further.
> 
> Cheers,
> Florian

Same issue for "eog" or "Eye of Gnome"

**Comment by rusty-snake** [comment]
#3277 @smitsohu 

@florianbegusch as a workaround `ignore read-only` should work.

shows `findmnt -R /home` any fuse mounts?

**Comment by smitsohu** [comment]
@florianbegusch There is a fix in master.

Would you mind sharing with us the `Warning: not remounting ...` messages?

**Comment by florianbegusch** [comment]
> findmnt -R /home

Sure.
```
$ findmnt -R /home
 /home                           /dev/mapper/VolGroup00-home               ext4   rw,rela
├─/home/foo/.local/share/containers
│                               /dev/mapper/VolGroup00-local_share_containers
│                                                                         ext4   rw,rela
└─/home/foo/VirtualBox_VMs

**Comment by florianbegusch** [comment]
Wow that was quick! Fixed 0d99824!
Thank you @smitsohu.
